pricing-page-exercise/salih/styles.css

```diff
@@ -0,0 +1,195 @@
+/* Global */
+
+*, ::after, ::before {
+    box-sizing: border-box;
+    font-family:sans-serif;
+}
+ul{
+    list-style-type: none;
+    padding: 0;
+    text-align: center;
+}
+.main li{
+    padding: 0.2rem 0;
+}
+h1{
+    margin: 0;
+    text-align: center;
+    margin-top: 5px;
+}
+button{
+    margin: 15px 10px;
+    min-width: 60%;
+    padding: 12px 0;
+}
+small{
+    font-weight: 100;
+    font-size: 0.5em;
+}
+.blue-button{
+    background-color: #007bff;
+    color: white;
+    border: none;
+    border-radius: 5px;
+    font-size: 1em;
+}
+
+.white-button{
+    color: #007bff;
+    background-color: white;
+    border: 1px solid #007bff;
+    border-radius: 5px;
+    font-size: 1em;
+}
+
+
+/*_____________________________________*/
+/*Navigation Bar*/
+
+.nav-container{
+    width: 100%;
+    min-height: 50px;
+    background-color: white;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    border-bottom: 1px solid #dee2e6;
+    box-shadow: 0 0.25rem 0.75rem rgba(0, 0, 0, .05);
+}
+.menu-container{
+    display: flex;
+    align-items: center;
+    min-width: 35%;
+    justify-content: center;
+    /* border: 1px solid red; */
+    flex-wrap: wrap;
+}
+.links{
+    display: flex;
+    min-width: 70%;
+    justify-content: space-between;
+    font-size: medium;
+}
+.menu{
+    display: flex;
+    justify-content: space-between;
+    min-width: 85%;
+    /* border: 1px solid red; */
+    list-style-type: none;
+    padding:0;
+    
+}
+.menu-item{
+    color: rgb(58, 58, 58);
+    padding: 1px;
+}
+.sign-up-button{
+    font-size: large;
+    border: 1px solid rgb(92, 92, 255);
+    
+    padding: 6px;
+    border-radius: 2px;
+}
+
+.company-name{
+    margin-left: 1rem;
+    font-size: x-large;
+}
+
+.blue{
+    color:rgb(92, 92, 255);
+}
+
+a:link{
+    text-decoration: none;
+}
+
+/*Pricing Text*/
+.pricing-container{
+    display: flex;
+    justify-content: center;
+    margin-top: 3rem;
+}
+.pricing-text{
+    text-align: center;
+    width: 40%;
+}
+.pricing-ptext{
+    font-size: 1.3em;
+    font-weight: 100;
+}
+
+/*______________________________________________*/
+/* The Cards are starting at this point */
+.main-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+.card-container{
+    width: 60%;
+    height: 50vh;
```

Comment:
Try not to set the height here as it may cause the layout to break on smaller screens when the content needs more space vertically

Reply:
I deleted the height value. Then footer got closer to card-container. I added a margin-botton to card-container.